Sources/Presentation/Presenter/AlertControllerFactory.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  AlertControllerFactory.swift
+//  Uppsala
+//
+//  Created by Suita Fujino on 2019/01/01.
+//  Copyright © 2019 Suita Fujino. All rights reserved.
+//
+
+import UIKit
+
+/**
+ A factory of `UIAlertController` with the default actions and the completion handler.
+ 
+ - Sample code:
+ ```
+ let factory = AlertControllerFactory(
+    title: "Update Notification",
+    message: "The new version of this app is now available!"
+ )
+ factory.actions = [.close(title: "Close"), .openAppStore(title: "App Store", url: url)]
+ let alertController = factory.build()
+ 
+ present(alertController, animated: true, completion: nil)
+ ```
+ */
+public class AlertControllerFactory {
+    
+    public typealias completionHandler = (DefaultAction, UIAlertAction) -> Void
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Scior/Uppsala/blob/efb3d7735090a484552a60a425de09570a0680fe/Sources/Presentation/Presenter/AlertControllerFactory.swift#L28"></span>Type name should start with an uppercase character: ‘completionHandler’</td>
    </tr>
  </tbody>
</table>